nbconvert/exporters/pdf.py

```diff
@@ -77,7 +77,7 @@ def run_command(self, command_list, filename, count, log_function):
         with open(os.devnull, 'rb') as null:
             stdout = subprocess.PIPE if not self.verbose else None
             for index in range(count):
-                p = subprocess.Popen(command, stdout=stdout, stdin=null)
+                p = subprocess.Popen(command, stdout=stdout, stdin=null, shell=True) #Shell=True okay since input is trusted.
```

Comment:
command is a list, and Popen(shell=True) only accepts when command is a string. If we do this, we will have to properly collapse the command-list into a command-string (with proper quoting, etc.)